### ISSUE TITLE
Usability of Lasair-ZTF

### ISSUE THREAD

**Issue by RoyWilliams** [lsst-uk/lasair#100]
Review web interface and functionality for Lasair-ZTF

**Comment by smarttgit** [comment]
I don't think we want to do this. Our focus now should be how we evolve to Lasair-LSST, not what we can do further for Lasair-ZTF

**Comment by RoyWilliams** [comment]
I would like to undertake some sort of overview of the Lasair web interface. There's a lot of squirrelly bits that I would hate to see grandfathered in to our new system.
